hw/ip/spi_host/doc/checklist.md

```diff
@@ -125,17 +125,17 @@ Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Not Started |
 Testbench     | [TB_GEN_AUTOMATED][]                  | Not Started |
 Tests         | [SIM_SMOKE_TEST_PASSING][]            | Not Started |
 Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Not Started |
-Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | Not Started |
+Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | DONE        |
```

Comment:
we don't have a standalone FPV TB for it, so it's N/A
```suggestion
Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | N/A        |
```

Reply:
oh shoot. too much copy paste - fixed